cleanup_k8s.py

```diff
@@ -12,6 +13,8 @@ def main():
         try:
             if ProviderChoice.GCE in providers:
```

Comment:
this won't work right anymore , you need loop on providers . if you will just add another "if" here . only first one will be processed than

Reply:
yes, you are rigth

Reply:
ah , you right . sorry